components/reusable-components/button/button.module.scss

```diff
@@ -20,6 +20,10 @@
   }
 }
 
+.smallFooterButton {
```

Comment:
better class naming here

Reply:
i didnt find where is used this and i delete it